cmd/net-state/main.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/julienschmidt/httprouter"
+
+	"github.com/storj/routes"
```

Comment:
just from an organizational perspective, a lot of different binaries are going to have routes. might make sense to have a subfolder inside of routes for netstate stuff

Reply:
also, i think this should be "storj.io/storj/routes" anyway, right? so "storj.io/storj/routes/netstate" maybe?

Reply:
personally I'd prefer "storj.io/storj/netstate/routes" but w/e